docs/ManualDeploymentGuideScalarDLAuditorOnAWS.md

```diff
@@ -64,23 +64,23 @@ so you need to add peering for internal communication between the Auditor, Ledge
     * Peering between Ledger and Client.
     * Peering between Auditor and Client.
 * Update route tables for VPC peering connection based on the [AWS official guide](https://docs.aws.amazon.com/vpc/latest/peering/vpc-peering-routing.html).
-* Add new Inbound and Outbound rule to restrict unwanted access to Ledger and Auditor.
-    * Add new Inbound rules to the Ledger Network ACLs to allow the Ledger Envoy LoadBalancer (e.g., 50051 and 50052 by default) access from the Auditor and Client.
+* Add new Inbound and Outbound rules to restrict unwanted access to Ledger and Auditor.
+    * Add new Inbound rules to Ledger Network ACLs to allow ephemeral ports (1024–65535) to access Ledger Envoy LoadBalancer (e.g., 50051 and 50052 by default) from the Auditor and Client.
```

Comment:
NACL is stateless, which means that it needs both an outbound rule and also an inbound rule for allowing traffic. The port range is always set for the destination port. Thus, when the application instance in Ledger VPC generates traffic for your instance in Auditor VPC, it will have the destination port as either 40051 or 40052, however, the source port would be a random port from the ephemeral port range of 1024 - 65535. So we need to allow ephemeral port range.